source/command_bus.coffee

```diff
@@ -7,19 +7,22 @@ class Space.messaging.CommandBus extends Space.Object
   }
 
   _handlers: null
+  _onSendHooks: null
 
   constructor: ->
     super
     @_handlers = {}
+    @_onSendHooks = []
 
-  send: (command, callback) ->
+  send: (command, callback, options={}) ->
     if @meteor.isServer
       # ON THE SERVER
       handler = @_handlers[command.typeName()]
-      if not handler?
+      hook(command) for hook in @_onSendHooks when !options.ignoreHooks
```

Comment:
…so if in testing we need to disable hooks (could be abstracted by BDD API):
```
app.configure({ messaging: { commandBus: { hooks: 'disable' }})
```
We need to work out a strategy for securing some configuration options, but that's inevitable anyway.